package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pillarwallet/pillarwallet-nodejs-sdk",
-  "version": "1.30.1",
+  "version": "1.30.2",
```

Comment:
You're adding a new thing here, the version should be `1.31.0`

Reply:
Already changed it, I saw it too :D

Reply:
:D